scripts/spellcheck.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Location of the typos configuration file
+TYPOS_CONFIG="_typos.toml"
+
+# Run typos command with appropriate options
+typos --config "$TYPOS_CONFIG" --write-changes
```

Comment:
I think we don't want to actually write changes. We just want to surface any errors for them to be manually corrected.